src/WCSimWCTrigger.cc

```diff
@@ -406,6 +409,210 @@ void WCSimWCTriggerBase::FillDigitsCollection(WCSimWCDigitsCollection* WCDCPMT,
 }
 
 
+void WCSimWCTriggerBase::FillDigitsCollectionNoTrigger(WCSimWCDigitsCollection* WCDCPMT, bool remove_hits, TriggerType_t save_triggerType)
```

Comment:
A bit of a naive question, but what are the differences between this & FillDigitsCollection()? They look very similar. Is there a way of combining them into a single method?